### PR TITLE
[PropertyAccess] __isset() returns true only if values are present

### PR DESCRIPTION
**Pull request by xabbuh**

### CODE REVIEW

**Comment by xabbuh** [comment]
ping @stof as you raised some concerns with the implementation from #19914

**Review by OskarStark — APPROVED** [review]

File: components/property_access.rst [hunk 1]
```diff
@@ -207,7 +207,7 @@ The ``getValue()`` method can also use the magic ``__get()`` method::
 
         public function __isset($id): bool
         {
-            return true;
+            return array_key_exists($id, $this->children);
```
Comment (nicolas-grekas) [inline components/property_access.rst:210]: isset should be used

Comment (javiereguiluz) [reply components/property_access.rst:210]: Changed while merging. Thanks.

**Comment by javiereguiluz** [comment]
Thanks Christian!

**Pull request merged**